pom.xml

```diff
@@ -154,6 +161,11 @@
       <version>${workflow.version}</version>
     </dependency>
     <dependency>
+      <groupId>org.jenkins-ci.plugins.workflow</groupId>
```

Comment:
Why did you add a dependency on the workflow-step-api plugin?  It already has that dependency for tests, and that is enough to compile and test the code.

I'd prefer you remove this before the change is merged to the 3.0.0-beta branch.